pkg/webhooks/mutator/virt-launcher-mutator_test.go

```diff
@@ -0,0 +1,159 @@
+package mutator
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	"github.com/kubevirt/hyperconverged-cluster-operator/api/v1beta1"
+	"github.com/kubevirt/hyperconverged-cluster-operator/controllers/commonTestUtils"
+	"github.com/kubevirt/hyperconverged-cluster-operator/pkg/util"
+)
+
+const (
+	hcoNamespace = HcoValidNamespace
+	podNamespace = "fake-namespace"
+)
+
+var _ = Describe("virt-launcher webhook mutator", func() {
```

Comment:
Please add error cases; e.g. no HyperConverged, negative ratio, zero ratio etc.

Reply:
Done!
Added tests for no HCO and invalid ratio. IMHO we don't need to test the library that parses the annotation as it's beyond this "unit's" scope.